File: server/dotnet/sdk-dotnet-example/Business/Domain/Common/Enums/StatusCategory.cs
namespace Business.Domain.Common.Enums;

public enum StatusCategory
{
    CREATED = 0,
    UNSUCCESSFUL = 1,
    PENDING_PAYMENT = 2,
    PENDING_MERCHANT = 3,
    PENDING_CONNECT_OR_3RD_PARTY = 4,
    COMPLETED = 5,
    REVERSED = 6, 
    REFUNDED = 7
}

File: server/java/src/main/java/com/onlinepayments/example/service/domain/common/enums/StatusCategory.java
package com.onlinepayments.example.service.domain.common.enums;

public enum StatusCategory {
    CREATED,
    UNSUCCESSFUL,
    PENDING_PAYMENT,
    PENDING_MERCHANT,
    PENDING_CONNECT_OR_3RD_PARTY,
    COMPLETED,
    REVERSED,
    REFUNDED
}
